fonts/notosans/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License

```

Comment:
The GPLv2 and Apache licenses are incompatible. (We already have one or two things in this situation, but I'd rather not dig us in deeper.) Can we bring this in e.g. via Composer or something?